website/templates/project/project.mako

```diff
@@ -112,10 +112,18 @@
                         <a href="${node['url']}register/${meta['name_no_ext']}">${meta['name_clean']}</a>
                     % endfor
                 % endif
-                <br />Date Created:
-                <span data-bind="text: dateCreated.local, tooltip: {title: dateCreated.utc}" class="date node-date-created"></span>
-                | Last Updated:
-                <span data-bind="text: dateModified.local, tooltip: {title: dateModified.utc}" class="date node-last-modified-date"></span>
+                % if node['is_registration']:
+                    <br />Date Registered:
+                    <span data-bind="text: dateRegistered.local, tooltip: {title: dateRegistered.utc}" class="date node-date-registered"></span>
+                    <br />Date Parent Project Created:
```

Comment:
"Parent Project" is potentially misleading--a registration isn't necessarily a "child" of the project it is registered from. I think the text in the blue alert and adding the Date Registered are enough to clarify the difference between registration vs. project and date created vs. date registered.

I am going to change the text back to "Date Created". 

@saradbowman Let me know if you disagree or have any further suggestions for improvement.